### PR TITLE
Add setting for basename

### PR DESCRIPTION
**Pull request by esagawe**
I added an optional options property to set a basename for the destination file (e.g. an id from a database).

### CODE REVIEW

File: src/thumbnail.js [hunk 1]
```diff
@@ -115,7 +115,8 @@ run = function(settings) {
 
     options = {
       srcPath: settings.source + '/' + image,
-      width: settings.width
+      width: settings.width,
+	  basename: settings.basename
```
Comment (honza) [inline src/thumbnail.js]: Nitpick: indentation

**Comment by honza** [comment]
This is a great idea! Would you mind documenting the new option?

**Comment by honza** [comment]
I'll make a release tonight. Thanks!

**Comment by esagawe** [comment]
Thanks for the new release!

**Pull request merged**